src/main/java/frc/robot/Orchestrator.java

```diff
@@ -230,12 +230,10 @@ public Command pullBack() {
    * @return The command to move the robot and intake.
    */
   public Command driveWhileIntaking() {
-    return Commands.parallel(
-        intake.intake().until(() -> RobotState.getInstance().hasNote),
+    return Commands.race(
+        intakeBasic(),
         Commands.run(
-                () -> drive.runVelocity(new ChassisSpeeds(Units.inchesToMeters(10), 0.0, 0.0)),
-                drive)
-            .withTimeout(2));
+            () -> drive.runVelocity(new ChassisSpeeds(Units.inchesToMeters(10), 0.0, 0.0)), drive));
```

Comment:
why no timeout?

Reply:
it will end when we intake a note (because it is a race cmd group)

Reply:
I think there should still be a timeout or the robot could end up driving all the way across the field.